t04_refactored.py

```diff
@@ -148,8 +148,65 @@ def team_4_adv(username, delay):
 
 
 def team_5_adv(username, delay):
-    pass
-    # TODO Add your code here
+    print("\n")
+
+    chalice = input(
+        "Hello intrepid explorer! You have stumbled upon The Flying Dutchman's lost treasure!" "\n"
+        "Before you lie 4 colored chalices, however, you can only choose one."
+        "Choose wisely! [Gold/Silver/Bronze/Brass]")
+    chalice.lower()
```

Comment:
This line isn't working because you don't save the result:

`chalice = chalice.lower()`